os/arch/arm/src/amebasmart/Kconfig

```
@@ -117,6 +129,22 @@ config AMEBASMART_I2S
 	default n
 
 if AMEBASMART_I2S
+config AMEBASMART_I2S0
+	bool "I2S_0"
+	default n
+if AMEBASMART_I2S0
+	config AMEBASMART_I2S0_MASTER
+	bool "IS20_MASTER"
```

Minor typo here and below,
```suggestion
	bool "I2S0_MASTER"
```